src/Service.Tests/GraphQLBuilder/Sql/StoredProcedureBuilderTests.cs

```diff
@@ -56,6 +56,8 @@ public class StoredProcedureBuilderTests
         [DataRow(typeof(DateTimeOffset), DATETIME_TYPE, "11/19/2012 10:57:11 AM -08:00", false, DisplayName = "DateTimeOffset")]
         [DataRow(typeof(TimeOnly), LOCALTIME_TYPE, "10:57:11.0000", false, DisplayName = "LocalTime")]
         [DataRow(typeof(byte[]), BYTEARRAY_TYPE, "AgQGCAoMDhASFA==", false, DisplayName = "Byte[]")]
+        [DataRow(typeof(Guid), UUID_TYPE, "f58b7b58-62c9-4b97-ab60-75de70793f66", false, DisplayName = "GraphQL UUID/ SystemType GUID")]
```

Comment:
The [HotChocolate docs](https://chillicream.com/docs/hotchocolate/v12/defining-a-schema/scalars#uuid-type) specifies 5 supported formats for UUID type. 

It would be good to have test cases with all the formats.

Reply:
I think this belongs in a separate work item because the tsql uuid type doesn't seem to support the other formats that HotChocolate specifies: https://learn.microsoft.com/en-us/sql/t-sql/data-types/uniqueidentifier-transact-sql?view=sql-server-ver16